themes/default/content/registry/packages/heroku/installation-configuration.md

```diff
@@ -10,7 +10,6 @@ The Pulumi Heroku provider is available as a package in all Pulumi languages:
 
 * JavaScript/TypeScript: [`@pulumiverse/heroku`](https://www.npmjs.com/package/@pulumiverse/heroku)
 * Python: [`pulumiverse_heroku`](https://pypi.org/project/pulumiverse_heroku/)
-* Go: [`github.com/pulumiverse/pulumi-heroku/sdk/go/heroku`](https://github.com/pulumiverse/pulumi-heroku/sdk/go/heroku)
```

Comment:
Request to fix: https://github.com/pulumiverse/pulumi-heroku/issues/65

Reply:
@ringods Fixed and new release published. Thanks for the hint.
https://github.com/pulumiverse/pulumi-heroku/releases/tag/v1.0.3